rest/src/main/java/fi/otavanopisto/pyramus/rest/controller/StudentContactLogEntryController.java

```diff
@@ -7,20 +7,26 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
+import fi.otavanopisto.pyramus.dao.students.StudentContactLogEntryCommentDAO;
 import fi.otavanopisto.pyramus.dao.students.StudentContactLogEntryDAO;
 import fi.otavanopisto.pyramus.domainmodel.students.Student;
 import fi.otavanopisto.pyramus.domainmodel.students.StudentContactLogEntry;
+import fi.otavanopisto.pyramus.domainmodel.students.StudentContactLogEntryComment;
 import fi.otavanopisto.pyramus.domainmodel.students.StudentContactLogEntryType;
+import fi.otavanopisto.pyramus.domainmodel.users.StaffMember;
 import fi.otavanopisto.pyramus.domainmodel.users.User;
 
 @Dependent
 @Stateless
 public class StudentContactLogEntryController {
   @Inject
   private StudentContactLogEntryDAO contactLogEntryDAO;
+  
+  @Inject
+  private StudentContactLogEntryCommentDAO contactLogEntryCommentDAO;
 
-  public StudentContactLogEntry createContactLogEntry(Student student, StudentContactLogEntryType type, String text, Date entryDate, String creator) {
-    StudentContactLogEntry contactLogEntry = contactLogEntryDAO.create(student, type, text, entryDate, creator);
+  public StudentContactLogEntry createContactLogEntry(Student student, StudentContactLogEntryType type, String text, Date entryDate, String creatorName, StaffMember creator) {
+    StudentContactLogEntry contactLogEntry = contactLogEntryDAO.create(student, type, text, entryDate, creatorName, creator);
```

Comment:
Minor issue but all these methods could simply return the value of the DAO call directly. No need for a throwaway variable within this method.